engine/engine/src/engine.cpp

```diff
@@ -1797,6 +1806,24 @@ namespace dmEngine
                 dmExtension::PostRender(&ext_params);
             }
 
+            if (engine->m_UseSwVSync && engine->m_UpdateFrequency > 0)
```

Comment:
The main condition:
* The backend doesn't support HW vsync
* The swap interval > 0
* The update frequency > 0

Reply:
Could you add it into manuals?